.travis.yml

```diff
@@ -29,8 +29,7 @@ matrix:
     - php: hhvm-nightly
 
 before_script:
-  - composer self-update
-  - composer install --no-interaction --prefer-source --dev
+  - composer install --no-interaction --prefer-source
```

Comment:
actually it's better to always do composer update & cache `~/.composer` like [nette/di](https://github.com/nette/di/blob/72854322c199f478d2e166bf28827e55202dfaac/.travis.yml) does it